omod/src/main/resources/apps/isantepluspatientdashboard_extension.json

```diff
@@ -78,5 +78,15 @@
         "url": "${project.parent.artifactId}/drugsHistory.page?patientId={{patient.uuid}}",
         "icon": "icon-bar-chart",
         "order": 7
+    },
+     {
+        "id": "${project.parent.groupId}.${project.parent.artifactId}.patientDashboard.secondColumnFragments.bmiGraph",
```

Comment:
now after adding such an entry, i always afterwards make the whole feature plug in/out of OpenMRS through a commit similar to https://github.com/IsantePlus/openmrs-module-isantepluspatientdashboard/commit/d5e4c8594d19b7bc5ef78e5174dc2a722cc911c2